packages/typography/src/useUpdatedBaseFontSize.ts

```diff
@@ -0,0 +1,13 @@
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { BaseFontSize } from '@leafygreen-ui/tokens';
+
+/**
+ * A wrapper around the hook useBaseFontSize that returns
+ * a value that is consistent with the updated base font sizes.
+ *
+ * Prefer using {@link useBaseFontSize} when LeafygreenProvider is updated to support the new values.
+ */
+export const useUpdatedBaseFontSize = (): BaseFontSize => {
```

Comment:
Can we document this in a way that clearly lets folks know that this API isn't sticking around in the long term? This will need to be a "public-ish" API so we can use it on other components.

Reply:
People won't reliably see a TS doc annotation, but might be good for us to get into the habit with it.